### DISCUSSION TITLE
Feldrandcontainer

### DISCUSSION THREAD

**Discussion by MacRama1** [Courseplay/Courseplay_FS22#2499] [Q&A]
Hallo CP Team,

kann CP sich selbstständig am Feldrand vom Feldrancontainer  mit Gülle/Gärresten befüllen, oder auch Kalk vom Überladewagen?
Ist das evtl. geplant oder übersehe ich die Funktion einfach nur.

Gruß Mac

**Comment by Tensuko** [comment]
Aktuell nicht, der Vorschlag steht aber unter Issues und könnte eventuell irgendwann als Funktion in CP landen.
